odxtools/diaglayer.py

```diff
@@ -636,9 +643,101 @@ def get_comparam(
 
         return cps[0]
 
-    def get_can_receive_id(self, protocol_name: Optional[str] = None) -> Optional[int]:
+    def get_can_mtu(self, protocol: Optional[Union[str, "DiagLayer"]] = None) -> Optional[int]:
+        """Return the maximum size of CAN frames that can be
```

Comment:
```suggestion
        """Return the maximum size of a CAN frame payload that can be
```

Reply:
In Linux Socket CAN, CAN MTU includes the CAN header, maybe stick to the standing naming of "Max Data Length"

Reply:
changed: e6459950662b002641d540ffcc471db993dd3e4b